File: pbworkspacewcl/src/app/notifiche-via-mail/commons/vo/notifiche-frequenze-vo.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

import { FrequenzaVO } from './frequenza-vo';
import { NotificaAlertVO } from './notifica-alert-vo';

export class NotificheFrequenzeVO {
    constructor(
        public frequenze: Array<FrequenzaVO>,
        public notificheAlert: Array<NotificaAlertVO>
    ) { }
}

File: pbworkspace/pbworkspace-jar/src/main/java/it/csi/pbandi/pbworkspace/integration/vo/NotificheFrequenzeVO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbworkspace.integration.vo;

import java.util.List;

public class NotificheFrequenzeVO {
	private List<FrequenzaVO> frequenze;
	private List<NotificaAlertVO> notificheAlert;

	public List<FrequenzaVO> getFrequenze() {
		return frequenze;
	}

	public void setFrequenze(List<FrequenzaVO> frequenze) {
		this.frequenze = frequenze;
	}

	public List<NotificaAlertVO> getNotificheAlert() {
		return notificheAlert;
	}

	public void setNotificheAlert(List<NotificaAlertVO> notificheAlert) {
		this.notificheAlert = notificheAlert;
	}
}
